interfaces/builtin/hidraw_test.go

```diff
@@ -198,6 +225,14 @@ SUBSYSTEM=="hidraw", SUBSYSTEMS=="usb", ATTRS{idVendor}=="ffff", ATTRS{idProduct
 	c.Assert(spec.Snippets(), HasLen, 1)
 	snippet = spec.Snippets()[0]
 	c.Assert(snippet, Equals, expectedSnippet2, Commentf("\nexpected:\n%s\nfound:\n%s", expectedSnippet2, snippet))
+
+	expectedSnippet3 := `IMPORT{builtin}="usb_id"
+SUBSYSTEM=="hidraw", SUBSYSTEMS=="usb", ATTRS{idVendor}=="abcd", ATTRS{idProduct}=="1234", ENV{ID_USB_INTERFACE_NUM}=="00", SYMLINK+="hidraw-myhidraw"`
+	spec = &udev.Specification{}
+	c.Assert(spec.AddPermanentSlot(s.iface, s.testUDev3), IsNil)
+	c.Assert(spec.Snippets(), HasLen, 1)
+	snippet = spec.Snippets()[0]
+	c.Assert(snippet, Equals, expectedSnippet3, Commentf("\nexpected:\n%s\nfound:\n%s", expectedSnippet3, snippet))
```

Comment:
Can you please drop `Commentf`, it's no longer needed (it's a legacy from the time those snippets were `[]byte`)

Reply:
Yeah, I'll drop it.